core/src/components/reorder/readme.md

```diff
@@ -13,8 +13,12 @@ Reorder is a component that allows an item to be dragged to change its order. It
 </ion-item>
 ```
 
-The position of the 
+## Properties
 
+| Property   | Attribute  | Description                            | Type      |
+| ---------- | ---------- | -------------------------------------- | --------- |
+| `slot`     | `slot`     | If `start`, the reorder icon will be shown before the label. If `end`, the reorder icon will be show after the label | `string` |
```

Comment:
We need to come up with a way to document slot across all components, instead of manually. Maybe we could have some kind of fake property that shows the available options for slot.

Reply:
How would we make the fake property? Just add a `@Prop` with some documentation?

Reply:
Let's remove this table for now since we'll be adding slot here: https://github.com/ionic-team/stencil/issues/1362

Reply:
👍 updated